### PR TITLE
Add jazzicon to delegate

### PR DESCRIPTION
**Pull request by alexkeating**
#### Description

- Add jazzicon if ENS avatar is missing
Screenshot from 2023-12-12 10-53-33

### CODE REVIEW

**Comment by alexkeating** [comment]
Current dependencies on/for this PR:
* main
  * **PR #174** <a href="https://app.graphite.dev/github/pr/ScopeLift/l2-flexible-voting-frontend/174?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/ScopeLift/l2-flexible-voting-frontend/174?utm_source=stack-comment).

**Review by garyghayrat — APPROVED** [review]

File: pages/[id]/delegate.tsx [hunk 1]
```diff
@@ -149,18 +151,20 @@ const Delegate: NextPage = () => {
...
                             />
+                          ) : (
+                            <Jazzicon diameter={24} seed={jsNumberForAddress(delegatee || '')} />
                           )}
-                          {delegateEnsName || delegatee}{' '}
+                          <p className="text-sm self-end">{delegateEnsName || delegatee}</p>
                           <InformationCircleIcon
                             id="delegate-address-information"
-                            className="h-4"
+                            className="h-4 self-start"
```
Comment (wildmolasses) [inline pages/[id]/delegate.tsx:167]:
```suggestion
                            className="h-4"
```

screenshot shows the info icon is no longer vertically aligned w the text; removing this is the fix. curious, did you not want the icon centered vertically anymore?

**Review by wildmolasses — APPROVED** [review]
one suggestion, otherwise lgtm

File: pages/[id]/delegate.tsx (hunk 1 above)
Comment (alexkeating) [reply pages/[id]/delegate.tsx:167]: Yes, good question. I thought it looked better higher, but if you guys think it looks worse I can change it back

Comment (wildmolasses) [reply pages/[id]/delegate.tsx:167]: my preference is centered vertically -- or next to the "Delgated to" rather than the address

Comment (alexkeating) [reply pages/[id]/delegate.tsx]: https://github.com/ScopeLift/l2-flexible-voting-frontend/pull/174/commits/a3d0c8939fa271e82b45d455ee75e5c8539798b1

**Pull request merged**